src/analytics/joblogs/useJobLogsAnalytics.ts

```diff
@@ -9,7 +9,8 @@ type Action =
   | { name: "Filter Job Logs"; filterBy: string }
   | { name: "Clicked lobster testlog url"; testId: string }
   | { name: "Clicked HTML testlog url"; testId: string }
-  | { name: "Clicked complete logs link"; taskId: string };
+  | { name: "Clicked complete logs link"; taskId: string }
+  | { name: "Clicked Parsley test log link"; testId: string };
```

Comment:
Maybe tracking buildId is more valuable? So we can get a sense of the amount of logs visited per build? Product would provide a better sense of the analytics they want.